src/Cache/BundleCacheClearer.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2016 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\ManagerBundle\Cache;
+
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\HttpKernel\CacheClearer\CacheClearerInterface;
+
+/**
+ * Clears the bundle cache.
+ *
+ * @author Kamil Kuzminski <https://github.com/qzminski>
+ */
+class BundleCacheClearer implements CacheClearerInterface
+{
+    /**
+     * @var Filesystem
+     */
+    private $filesystem;
+
+    /**
+     * Constructor.
+     *
+     * @param Filesystem $filesystem
+     */
+    public function __construct(Filesystem $filesystem)
+    {
+        $this->filesystem = $filesystem;
```

Comment:
I am not sure, @Toflar recommended the below as the similar solution so I just based it on that. Should we change it in the core as well then?

https://github.com/contao/core-bundle/blob/master/src/Cache/ContaoCacheClearer.php#L33